tests/core/pyspec/eth2spec/test/context.py

```diff
@@ -29,10 +27,15 @@ def reload_specs():
 ConfigName = NewType("ConfigName", str)
 
 PHASE0 = SpecForkName('phase0')
-PHASE1 = SpecForkName('phase1')
 ALTAIR = SpecForkName('altair')
 
-ALL_PHASES = (PHASE0, PHASE1, ALTAIR)
+# Experimental phases (not included in default "ALL_PHASES"):
+MERGE = SpecForkName('merge')
+SHARDING = SpecForkName('sharding')
+CUSTODY_GAME = SpecForkName('custody_game')
+DAS = SpecForkName('das')
+
+ALL_PHASES = (PHASE0, ALTAIR)
```

Comment:
Doesn't MERGE pass all PHASE0 tests? I think we should include it in CI

Reply:
Let's do so in another PR before this one grows too large